### ISSUE TITLE
A11y_Dynamics 365 Relationship Insights (Relationship Assistant)_Insight Cards_Screen Reader: In scan mode NVDA is not reading the name for "All Security Roles"

### ISSUE THREAD

**Issue by InduPriya1805** [nvaccess/nvda#10252]
**Environment Details:**
URL:​https://zidadach0628.crm10.dynamics.com/main.aspx?appid=5243f9a2-f5bf-e911-a97b-000d3a4e9aa0&pagetype=control&controlName=MscrmControls.UnifiedConfigCustomControl.UnifiedConfigCustomControl
User Name: tiptesttenant@tipregion.onmicrosoft.com
Server version: 9.1.0000.8614
Client version: 1.3.2257-190830-002936
Browser Details: Chrome76.0.3809.132​

**Repro Steps:​**
1)Turn on NVDA in scan mode
2) Hit the below url​
"​https://zidadach0628.crm10.dynamics.com/main.aspx?appid=5243f9a2-f5bf-e911-a97b-000d3a4e9aa0&pagetype=control&controlName=MscrmControls.UnifiedConfigCustomControl.UnifiedConfigCustomControl" and login with appropriate credentials.
3)Tab till "Insight Cards" tab and press enter
4) While navigating in scan mode verify  NVDA is reading the name for "All Security Roles" or not

**Actual Result:​**
In scan mode NVDA is not reading the name for "All Security Roles"

**Expected Result:​**
In scan mode NVDA should read the name for "All Security Roles"

**User Impact:​**
Screen Reader users will not get to know the name of the controls if NVDA does not announce it
​
**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
![Bug_10252_In scan mode NVDA not reading name](https://user-images.githubusercontent.com/42638450/65326118-5823d500-dbce-11e9-8172-71c186dab5fe.png)

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
